src/spikeinterface/sorters/external/kilosort4.py

```diff
@@ -222,36 +244,22 @@ def _run_from_folder(cls, sorter_output_folder, params, verbose):
             probe_name=probe_name,
             data_dir=data_dir,
             results_dir=results_dir,
+            bad_channels=bad_channels,
         )
 
-        if version.parse(cls.get_sorter_version()) >= version.parse("4.0.12"):
-            ops = initialize_ops(
-                settings=settings,
-                probe=probe,
-                data_dtype=recording.get_dtype(),
-                do_CAR=do_CAR,
-                invert_sign=invert_sign,
-                device=device,
-                save_preprocesed_copy=save_preprocessed_copy,  # this kwarg is correct (typo)
-            )
-        else:
-            ops = initialize_ops(
-                settings=settings,
-                probe=probe,
-                data_dtype=recording.get_dtype(),
-                do_CAR=do_CAR,
-                invert_sign=invert_sign,
-                device=device,
-            )
+        ops = initialize_ops(
+            settings=settings,
+            probe=probe,
+            data_dtype=recording.get_dtype(),
+            do_CAR=do_CAR,
+            invert_sign=invert_sign,
+            device=device,
+            save_preprocessed_copy=save_preprocessed_copy,  # this kwarg is correct (typo)
```

Comment:
Just to doublecheck. As of the most recent version this kwarg is corrected or not?

Reply:
Yes - corrected since 4.0.13